tools/Custom/HttpMessageLogFormatter.cs

```diff
@@ -92,11 +91,13 @@ private static string FormatString(string content)
         {
             try
             {
-                if (CloudException.IsJson(content))
+                content = content.Trim();
+                if ((content.StartsWith("{") && content.EndsWith("}")) || // object
+                    (content.StartsWith("[") && content.EndsWith("]"))) // array
                 {
                     return JsonConvert.SerializeObject(JsonConvert.DeserializeObject(content), Formatting.Indented);
                 }
-                if (CloudException.IsXml(content))
+                if (content.StartsWith("<"))
```

Comment:
Wouldn't a header check, where we check the response content type returned from the service, be more reliable?